t2.cpp

```diff
@@ -0,0 +1,6 @@
+sdjfsjsdjfsd;
+fsdfsdfsf;
+sdfdsfdsf;
+sdfsdfsdf;
+dsfsdfs;
+dsf;
```

Comment:
variable 'dsf' can be made static or moved into an anonymous namespace to enforce internal linkage [misc-use-internal-linkage,-warnings-as-errors]